sidechain/consensus/aura/src/mock.rs

```diff
@@ -85,6 +88,30 @@ impl<SB: SidechainBlockT> SidechainSystemExt<SB> for StateMock<SB> {
 	fn set_last_block(&mut self, block: &SB) {
 		self.last_block = Some(block.clone())
 	}
+
+	fn get_block_number(&self) -> Option<BlockNumber> {
```

Comment:
these should be implemented inside the stf scope I guess?
Can you implement them as indicated in the issue?

Reply:
Ok, so you also need specific access to these functions from the STF directly, but not (only) access to the storage value from inside the pallet?

My idea was to cleanly separate STF and sidechain. STF would be a simple extension to the sidechain and have access to the methods the sidechain db provides. I am not certain if this can be easily done with before the STF is refactored.

I will quickly think about it. If it is not possible, I will replicate the code inside the STF.